yb-voyager/cmd/exportData.go

```diff
@@ -805,6 +805,13 @@ func checkDataDirs() {
 		if err != nil {
 			utils.ErrExit("Failed to truncate tables in metadb: %s", err)
 		}
+		if changeStreamingIsEnabled(exportType) {
```

Comment:
Do we really need this check? 
`ClearMigrationState` should be able to handle both offline/live migration types

Reply:
Yes right now we need this check because we are only dropping the Log mining flush table in this function and that is also just for Oracle, so log mining flush can only be dropped in case of live migration. 
Ideally, the query should be appropriate to drop the table only if exists but the direct if exists clause doesn't exist in Oracle so need to modify the query. So I have changed the query to handle if an exists case after which this condition is not required